robot.py

```diff
@@ -140,3 +141,9 @@ def steering_angle(self, speed, sharpness, angle):
             self.tank.drive(speed, sharpness)
         self.tank.stop()
     
+    
+    def read_calibrate(self):
+        file_handler = open("Calibration.txt", "r")
```

Comment:
It is better if we account for the case where this file is missing and fall back to a default value (e.g., `12`)